keps/sig-scheduling/3521-pod-scheduling-readiness/README.md

```diff
@@ -965,8 +965,10 @@ Pick one more of these and delete the rest.
 Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
 implementation difficulties, etc.).
 -->
+Yes, we will need a new metric `pod_scheduling_excluding_preenqueue_duration_seconds` to explicitly differentiate whether the time spent during `PreEnqueue` is included or excluded.
```

Comment:
@helayoty let's get https://github.com/kubernetes/kubernetes/pull/118049 merged and cherrypick PRs done first, and then update this KEP.

Reply:
Done.